apps/frontend-manage/src/components/sessions/creation/LiveSessionWizard.tsx

```diff
@@ -376,71 +386,124 @@ function StepTwo(props: StepProps) {
+              ]}
+              data={{ cy: 'select-multiplier' }}
+              className={{ tooltip: 'z-20' }}
+            />
+            <ErrorMessage
+              name="multiplier"
+              component="div"
+              className="text-sm text-red-400"
+            />
+          </div>
+        </div>
+      )}
+      <div className="flex flex-col gap-4">
+        <div>
+          <FormikSwitchField
+            name="isConfusionFeedbackEnabled"
+            label={t('shared.generic.feedbackChannel')}
+            tooltip={t('manage.sessionForms.liveSessionFeedbackChannel')}
+            standardLabel
+            data={{ cy: 'set-gamification' }}
             className={{ tooltip: 'z-20' }}
           />
           <ErrorMessage
-            name="courseId"
+            name="isConfusionFeedbackEnabled"
             component="div"
             className="text-sm text-red-400"
           />
         </div>
-      )}
-      <div>
-        <FormikSwitchField
-          disabled={values.courseId === ''}
-          name="isGamificationEnabled"
-          label={t('shared.generic.gamification')}
-          tooltip={t('manage.sessionForms.liveSessionGamification')}
-          standardLabel
-          data={{ cy: 'set-gamification' }}
-          className={{ tooltip: 'z-20' }}
-        />
-        <ErrorMessage
-          name="isGamificationEnabled"
-          component="div"
-          className="text-sm text-red-400"
-        />
-      </div>
-      <div className="flex flex-row items-center gap-4">
-        <FormikSelectField
-          disabled={!values.isGamificationEnabled}
-          name="multiplier"
-          label={t('shared.generic.multiplier')}
-          tooltip={t('manage.sessionForms.liveSessionMultiplier')}
-          placeholder={t('manage.sessionForms.multiplierDefault')}
-          items={[
-            { label: t('manage.sessionForms.multiplier1'), value: '1' },
-            { label: t('manage.sessionForms.multiplier2'), value: '2' },
-            { label: t('manage.sessionForms.multiplier3'), value: '3' },
-            { label: t('manage.sessionForms.multiplier4'), value: '4' },
-          ]}
-          data={{ cy: 'select-multiplier' }}
-          className={{ tooltip: 'z-20' }}
-        />
-        <ErrorMessage
-          name="multiplier"
-          component="div"
-          className="text-sm text-red-400"
-        />
+        <div>
+          <FormikSwitchField
+            name="isLiveQAEnabled"
+            label={t('shared.generic.liveQA')}
+            tooltip={t('manage.sessionForms.liveSessionLiveQA')}
+            standardLabel
+            data={{ cy: 'set-gamification' }}
+            className={{ tooltip: 'z-20' }}
+          />
+          <ErrorMessage
+            name="isLiveQAEnabled"
+            component="div"
+            className="text-sm text-red-400"
+          />
+        </div>
+        {values.isLiveQAEnabled && (
```

Comment:
What do you think of simply disabling this functionality in case Live Q&A is disabled? This way, professors can directly see what settings can be preset and this behaviour would also be more consistent (e.g., Gamification & Multiplier in Session Creation and  Live Q&A and Moderation in Lecturer Cockpit)

Reply:
You mean that the setting cannot be changed if the live QA is not enabled? This should already be implemented, if I remember it correctly. That the default value for this is set to true, even if the live QA is disabled, was a deliberate decision, which was taken quite some time ago :)

Reply:
Sorry, I was not that clear😅 What I really meant was instead of hiding the moderation switch if Live QA is not enabled, we can simply disable the switch as was done the same in the mentioned examples🤔

Reply:
Ah right, I thought it would be smart to hide it for disabled live QA as the default option of the moderation is true and the lecturer cannot disable it then initially...

Reply:
Implementation has been adapted - switches are always shown, disabling is highlighted more explicitly in new design-system version, thanks for the feedback :)